src/controllers/logoutController.ts

```diff
@@ -0,0 +1,40 @@
+import { Blacklist } from "../database/models/blacklist";
+import { HttpException } from "../utils/http.exception";
+import { Request, Response } from "express";
+import { v4 as uuidv4 } from "uuid";
+
+const logout = async (req: Request, res: Response) => {
+	try {
+		const token = req.header("Authorization")?.split(" ")[1];
```

Comment:
Why are you still getting token in url, I hope @leandreAlly  gave you feedback in last standup.